README.md

```diff
@@ -1,78 +1,65 @@
-# Grafana Plugin Repository
+# Community plugins
 
-This repository lists all officially supported Grafana plugins. Plugins in this repository are listed on [Grafana.com](https://grafana.com/grafana/plugins), and can be [installed](https://grafana.com/docs/grafana/latest/plugins/installation/) locally using [Grafana CLI](https://grafana.com/docs/grafana/latest/administration/cli/#plugins-commands) or by [Grafana Cloud](https://grafana.com/products/cloud/) users.
+We created this repository in 2016 to accept plugin submissions from the Grafana community. 5 years and more than 150 published plugins later, it's clear that community plugins have been instrumental in the success of Grafana. We'd like to make the process of submitting plugins even easier.
```

Comment:
Just a note that this paragraph will require maintenance on a yearly basis. One simple way to address this would be to be to change "5 years" to "Five-plus years," but I would avoid this kind of verbiage all together. 

NIT: Assuming we want to stay in the first person, and keep an informal voice, I might suggest: "We originally created this repository years ago to accept plugin submissions from the Grafana community. With more than 150 plugins now published, it has become obvious that the community plugins are instrumental in Grafana's success. Due to this, we've make the process of submitting plugins easier."